configuration/backend_configuration/ampathforms/procedure-history-form.json

```
@@ -0,0 +1,194 @@
+              "required": true
+            },
+            {
+              "type": "obs",
+              "questionOptions": {
+                "rendering": "datetime",
+                "concept": "160715AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+              },
+              "id": "procedureDate",
+              "label": "Date/time procedure performed",
+              "datePickerFormat": "both",
+              "validators": [
+                {
+                  "type": "date",
+                  "allowFutureDates": "false"
+                }
+              ]
+            },
+            {
+              "type": "obs",
+              "questionOptions": {
+                "rendering": "datetime",
+                "concept": "167132AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+              },
+              "id": "procedureEndDate",
+              "label": "Procedure end date/time",
+              "datePickerFormat": "both",
+              "validators": [
+                {
+                  "type": "date",
+                  "allowFutureDates": "true"
+                }
+              ]
+            },
+            {
+              "type": "obs",
+              "questionOptions": {
+                "rendering": "text",
+                "concept": "163049AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA"
+              },
+              "id": "procedureSite",
+              "label": "Procedure site"
+            },
+            {
+              "type": "obs",
+              "questionOptions": {
+                "rendering": "select",
+                "concept": "160721AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                "answers": [
+                  {
+                    "concept": "160717AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Procedure outcome satisfactory to patient"
+                  },
+                  {
+                    "concept": "160718AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Procedure outcome satisfactory to physician"
+                  },
+                  {
+                    "concept": "160719AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Procedure outcome unsatisfactory to patient"
+                  },
+                  {
+                    "concept": "160720AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                    "label": "Procedure outcome unsatisfactory to physician"
+                  }
+                ]
+              },
+              "id": "procedureOutcome",
+              "label": "Procedure outcome"
+            },
+            {
+              "type": "obs",
+              "questionOptions": {
+                "rendering": "number",
+                "concept": "165929AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+                "min": "1",
+                "max": "100"
+              },
+              "id": "procedureDuration",
+              "label": "Duration of procedure"
+            },
+            {
+              "type": "obs",
+              "id": "durationUnit",
```

```suggestion
              "id": "procedureDurationUnit",
```

For consistency with other IDs.